packages/ui-extensions/src/surfaces/admin/components/Chip/Chip.doc.ts

```diff
@@ -29,6 +29,183 @@ const data: AdminReferenceEntityTemplateSchema = {
+                  code: './examples/order-status.html',
+                  language: 'preview',
+                },
+              ],
+            },
+          },
+          {
+            description:
+              'Inventory tracking indicators for product availability.',
+            codeblock: {
+              title: 'Inventory status',
+              tabs: [
+                {
+                  code: './examples/inventory-status.html',
+                  language: 'preview',
+                },
+              ],
+            },
+          },
+          {
+            description: 'Payment state indicators for transaction tracking.',
+            codeblock: {
+              title: 'Payment status',
+              tabs: [
+                {
+                  code: './examples/payment-status.html',
+                  language: 'preview',
+                },
+              ],
+            },
+          },
+          {
+            description:
+              'Customer categorization labels for segmentation and personalization.',
+            codeblock: {
+              title: 'Customer tags',
+              tabs: [
+                {
+                  code: './examples/customer-tags.html',
+                  language: 'preview',
+                },
+              ],
+            },
+          },
+          {
+            description:
+              'Shipping and delivery option indicators for order management.',
+            codeblock: {
+              title: 'Shipping methods',
+              tabs: [
+                {
+                  code: './examples/shipping-methods.html',
+                  language: 'preview',
+                },
+              ],
+            },
+          },
+          {
+            description:
+              'Discount and promotion indicators for pricing and marketing.',
+            codeblock: {
+              title: 'Discount labels',
+              tabs: [
+                {
+                  code: './examples/discount-labels.html',
+                  language: 'preview',
+                },
+              ],
+            },
+          },
+          {
+            description:
+              'Product variation attributes for inventory management.',
+            codeblock: {
+              title: 'Product variants',
+              tabs: [
+                {
+                  code: './examples/product-variants.html',
+                  language: 'preview',
+                },
+              ],
+            },
+          },
+          {
+            description: 'Product collection labels for catalog organization.',
+            codeblock: {
+              title: 'Collections',
+              tabs: [
+                {
+                  code: './examples/collections.html',
+                  language: 'preview',
+                },
+              ],
+            },
+          },
+          {
+            description:
+              'Demonstrates automatic text truncation for long content.',
```

Comment:
This example doesn't seem to show properly in the preview. I'm not sure what we're trying to demonstrate here, but it seems like we're not demonstrating it well.

<img width="653" height="193" alt="image" src="https://github.com/user-attachments/assets/c3b7c09f-7748-4fca-9bce-262ff6a3b7cb" />

Reply:
now it should be properly constrained:

<img width="1304" height="1124" alt="Screenshot 2025-09-30 at 1 36 31 PM" src="https://github.com/user-attachments/assets/f844e5e9-104f-4e56-9284-11110140a6c4" />